__tests__/exchanges/gmocoin-client.spec.js

```diff
@@ -0,0 +1,81 @@
+const { testClient } = require("../test-runner");
+const GMOCoinClient = require("../../src/exchanges/gmocoin-client");
+
+testClient({
+  clientFactory: () => new GMOCoinClient(),
+  clientName: "GMOCoinClient",
+  exchangeName: "GMOCoin",
+  markets: [
+    {
+      id: "BTC",
+      base: "BTC",
+      quote: "JPY",
+    },
+    {
+      id: "ETH",
+      base: "ETH",
+      quote: "JPY",
+    },
+    {
+      id: "BCH",
+      base: "BCH",
+      quote: "JPY",
+    },
+    {
+      id: "LTC",
+      base: "LTC",
+      quote: "JPY",
+    },
+    {
+      id: "XRP",
+      base: "XRP",
+      quote: "JPY",
+    },
+  ],
+
+  testConnectEvents: false,
```

Comment:
These should be true. We should test the client events to make sure the client is functioning correctly.